src/bundle/Resources/public/js/alloyeditor/src/buttons/ez-btn-customtag.js

```diff
@@ -18,6 +18,7 @@ export default class EzBtnCustomTag extends EzWidgetButton {
             return <ButtonComponent
                 createNewTag="true"
                 values={this.values}
+                content={this.content}
```

Comment:
Where does the `this.content` come from?

Reply:
This notation looks weird in React component.

Reply:
Same place than `this.values`, afaik, I've tried to imitate what was done with it. To my knowledge, it is initialized there: https://github.com/ezsystems/ezplatform-admin-ui/blob/custom_tag_content_edit/src/bundle/Resources/public/js/alloyeditor/src/buttons/ez-btn-customtag-update.js#L10-L13. But as I've said: trial and error :)

Reply:
@tischsoic if you have free resources, you can spend time on it. But keep in mind, that the Alloy Editor widgets code is our legacy code from v1. Some things might not work as expected.

Reply:
Note that the initialization of `values` and `content` differ. `values` requires an actual initialization, as it takes values from server side config (default value, required, type...). `content` is not initialized to a value. It receives what's in the DOM when editing an existing tag, and is set to whatever is entered in the textarea.

Besides that, your conversation is beyond my un-knowledge. Any guidance or change is very welcome.